### PR TITLE
SEOImageUpdates: User Story 1463880, Small PR 1

### PR DESCRIPTION
**Pull request by damabe**
Contributes to #10839

### CODE REVIEW

**Comment by mairaw** [comment]
Do these smaller PRs replace #10947 @damabe?

File: docs/framework/deployment/reducing-system-restarts.md [hunk 1]
```diff
@@ -20,8 +20,7 @@ The [!INCLUDE[net_v45](../../../includes/net-v45-md.md)] installer uses the [Res
   
  If the Restart Manager detects a situation that will require a system restart even if running apps are closed, the message is not displayed.  
   
- ![Close Application Dialog](../../../docs/framework/deployment/media/closeapplicationdialog.png "CloseApplicationDialog")  
-Prompt for closing .NET Framework apps that are in use  
+ ![Screenshot that shows the Close Application dialog.](./media/reducing-system-restarts/close-application-dialog.png)
```
Comment (mairaw) [inline docs/framework/deployment/reducing-system-restarts.md]:
```suggestion
 ![The Close Application dialog listing the programs currently running.](./media/reducing-system-restarts/close-application-dialog.png)  
```

File: docs/framework/wcf/feature-details/managing-claims-and-authorization-with-the-identity-model.md [hunk 2]
```diff
@@ -84,11 +84,11 @@ Authorization is the process of determining which entities have permission to ch
   
  The following figure shows an example of three sets of claims where one set of claims has, as its issuer, another set of claims, which in turn has the System claim set as its issuer. Therefore, sets of claims form a hierarchy that may be arbitrarily deep.  
   
- ![Managing claims and authorization](../../../../docs/framework/wcf/feature-details/media/claimshierarchy.gif "claimshierarchy")  
+ ![Sets of claims within the hierarchy.](./media/managing-claims-and-authorization-with-the-identity-model/claims-sets-hierarchy.gif)  
   
  Multiple sets of claims may have the same issuing claim set, as illustrated in the following figure.
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/managing-claims-and-authorization-with-the-identity-model.md]:
```suggestion
 Multiple sets of claims may have the same issuing claim set, as illustrated in the following figure:
 
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @damabe. A few comments before merging this one.

**Comment by damabe** [comment]
These do replace the large PR. I've got one more to create to cover all the updated file. I'll do that one in a bit and @ mention you when I'm done.

**Comment by damabe** [comment]
@mairaw Feedback incorporated. Updated CATS report:

https://contentqacats.azurewebsites.net/CheckResults/2019051500056/Report/

**Comment by mairaw** [comment]
Breaking it down helps a lot @damabe. Thanks!

**Review by mairaw — APPROVED** [review]

**Pull request merged**